proto/v1beta1/vulnerability.proto

```diff
@@ -238,7 +238,13 @@ message Details {
 
   VexAssessment vex_assessment = 10;
 
-  // Next free ID is 11.
+  // The cvss v2 score for the vulnerability.
+  CVSS cvssv2 = 11;
```

Comment:
Could you use snake case for field names? i.e. cvss_v2 and cvss_v3.

Reply:
Thanks. Updated.